komapper-core/src/main/kotlin/org/komapper/core/type/BlobByteArray.kt

```diff
@@ -0,0 +1,8 @@
+package org.komapper.core.type
+
+@JvmInline
+value class BlobByteArray(val value: ByteArray) {
+    override fun toString(): String {
+        return value.contentToString()
```

Comment:
The `toString()` method of `BlobByteArray` is called when logging.
Using `contentToString()` could produce very long log entries if the byte array is large.
Since we rarely need to log the full byte array, I think calling `toString()` would be better.